packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import Select from 'muicss/lib/react/select'
+import { BaseInput } from '../util/Input'
+
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { Upload } from '../util/Upload'
+import { useControlRef } from '../util/ControlRef'
+import { TogglableDropdown } from '../util/TogglableDropdown'
+
+const options = {
+  driversLicenseNumber: 'Driver\'s License Number',
+  copyOfPhotoId: 'Copy of Photo ID',
+} as const
+
+export const Kansas = () => {
+  const idNumberRef = useControlRef<Select>()
```

Comment:
@ludanin: since this is now a controlled form, can we still use ref's correctly?